attributes/defaults.rb

```diff
@@ -18,17 +18,29 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Attribute that defines whether MariaDB or MySQL should be used
-default['mysqld']['use_mariadb'] = true
+# Attribute that defines whether MariaDB, MySQL or Percona should be used
+default['mysqld']['db_install'] = 'mariadb'
 
 # Default packages to install
 default['mysqld']['mysql_packages'] = %w(mysql-server)
 default['mysqld']['mariadb_packages'] = %w(mariadb-server)
 default['mysqld']['mariadb_galera_packages'] = %w(mariadb-galera-server)
 
 # MariaDB repository configuration
-default['mysqld']['repository']['version'] = '10.1'
-default['mysqld']['repository']['mirror'] = 'http://ftp.hosteurope.de/mirror/mariadb.org/repo'
+default['mysqld']['repository']['mariadb']['version'] = '10.1'
+default['mysqld']['repository']['mariadb']['mirror'] = 'http://ftp.hosteurope.de/mirror/mariadb.org/repo'
+
+# Percona repository configuration
+default['mysqld']['repository']['percona']['version'] = '5.7'
+default['mysqld']['repository']['percona']['mirror'] = 'http://repo.percona.com/apt'
+
+if node['mysqld']['repository']['percona']['version'] == '5.7'
+  default['mysqld']['percona_packages'] = %w(percona-server-server-5.7)
```

Comment:
I think we should also migrate the repository attributes to be consistent with the new hierarchy:

```ruby
default['mysqld']['mariadb']['version'] = 10.1 # We shouldn't use a string here, I suppose
default['mysqld']['mariadb']['repository'] = 'http://ftp.hosteurope.de/mirror/mariadb.org/repo'
```

The `version` attribute can then be used later to determine the correct `pwd_col`.